homeassistant/helpers/entity_platform.py

```diff
@@ -343,6 +343,14 @@ def add_entities(self, new_entities, update_before_add=False):
                 known_object_ids=self.entities.keys(),
             )
 
+            if not entry:
```

Comment:
We should not add this, instead we should leverage the `entry.disabled` check below.